AudioComments/AudioComments/AudioCommentView/AudioCommentView.swift

```diff
@@ -0,0 +1,161 @@
+//
+//  AudioCommentView.swift
+//  AudioComments
+//
+//  Created by Shawn Gee on 5/5/20.
+//  Copyright © 2020 Swift Student. All rights reserved.
+//
+
+import UIKit
+
+protocol AudioCommentViewDelegate: AnyObject {
+    func startRecording(for audioCommentView: AudioCommentView)
+    func stopRecording(for audioCommentView: AudioCommentView)
+    func startPlayback(for audioCommentView: AudioCommentView)
+    func pausePlayback(for audioCommentView: AudioCommentView)
+    func scrubPlayback(to location: Float, for audioCommentView: AudioCommentView)
+    
+    func sendAudioComment(for audioCommentView: AudioCommentView)
+    func sendTextualComment(with text: String, for audioCommentView: AudioCommentView)
+}
+
+class AudioCommentView: UIView {
```

Comment:
This view was getting squished for me on an iPhone 11 Pro, where the right-hand buttons were tiny; might be good to (eventually) adjust things so that the slider is resized rather than those buttons.